doc/python/probe_sample.rst

```diff
@@ -5,14 +5,14 @@ Cable cell probing and sampling
 
 .. module:: arbor
 
-Cable cell probe addresses are defined analogously to their counterparts in
+Cable cell probeset addresses are defined analogously to their counterparts in
 the C++ API (see :ref:`cablecell-probes` for details). Sample data recorded
 by the Arbor simulation object is returned in the form of a NumPy array,
 with the first column holding sample times, and subsequent columns holding
 the corresponding scalar- or vector-valued sample.
```

Comment:
You know me ... a graphic? A crude approximation
```
O-------------         Cell morphology
^    ^    ^            Probe locset
|    |    |
L0 L1 L2               Locations, eg (cable 0 0.23)
U0 V0 W0               Values, one column per Location
U1                     one entry per sampling time point
U2
...
```

Reply:
Thanks for adding the graphic, but I think we could be even clearer by splitting it in two
1. Showing the relation of handle, probeset, probes, and their locations
2. Showing the relation between the sampled data and the probeset

Sorry for being picky here, but we inflict multiple layers of callbacks and handles on our
users (for good reasons), so giving them a hand is needed. 

That said, I made one of the images, so maybe that plus the more schematic one you added
is enough. Tell me what you think.

Also, since I cannot comment on full files, feedback for the graphic here
1. Typo: experession
2. Did you check array format? I thought I remember columns of time, value. As shown it indicates rows. Maybe this would be clearer if shown as a table.
3. The `->` relation is unclear
  - handle -> probeset: _obtain_ a probeset from simulation
  - probeset -> probe, probe -> data|meta: ownership
  - meta -> locset, data -> ndarray: is a

On 3. I would suggest showing 'has a' and 'is a' relations like this; example for probe
```
+---------------+
| Probe         |
| meta: locset  | 
| data: ndarray |
+---------------+
```
The handle -> probeset -> probe could remain.